api/build-tools/json.lua

```diff
@@ -0,0 +1,400 @@
+--
```

Comment:
Not sure if this file has the license issue, cc @moonming

Reply:
@nic-chen where is this file come from? 
We need a list of those copied files and source addresses in order to check license

Reply:
@moonming 
It's from https://github.com/rxi/json.lua
I think we could resolve license issues together later.